app/src/main/java/com/grelp/grelp/activities/MainActivity.java

```diff
@@ -46,13 +50,15 @@
     private static final int PERMISSION_REQUEST_CODE = 1;
     private final static int CONNECTION_FAILURE_RESOLUTION_REQUEST = 9000;
 
+    private LocationRequest mLocationRequest;
+
     private DealListFragment dealListFragment;
     private DealMapFragment dealMapFragment;
 
     private GoogleApiClient mGoogleApiClient;
-    private static final long UPDATE_INTERVAL = 60000 * 60;  /* 60 secs */
+    private static final long UPDATE_INTERVAL = 1000 * 30;  /* 30 secs */
```

Comment:
True, but I also added the minimum distance, so you shouldn't get an update unless you move more than 200m. I was worried that if the first update is null, it might be ages before we get another one - although it seems we get null only from the emulator.